config/crd/external/user.openshift.io_users.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
-  creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.13.0
```

Comment:
do we want to upgrade controller-gen version? 
can keep v0.9.2?

Reply:
I will revert it

Reply:
I will revert it